packages/desktopjs/src/Default/default.ts

```diff
@@ -386,9 +386,9 @@ export namespace Default {
                             layout.windows.push(
                                 {
                                     name: window.name,
-                                    url: (nativeWin && nativeWin.location) ? nativeWin.location.toString() : undefined,
+                                    url: nativeWin?.location?.toString(),
                                     id: window.id,
-                                    bounds: { x: nativeWin.screenX, y: nativeWin.screenY, width: nativeWin.innerWidth, height: nativeWin.innerHeight },
+                                    bounds: await window.getBounds(),
```

Comment:
This is a breaking change as it looks like getBounds is based on outerWidth/outerHeight while layout is based on innerwidth/innerHeight.

Reply:
I see, you're right. I'll have to do this in the container specific implementation then.

Reply:
Or we have a different bug with the conflict.  I'm curious as to whether this was intentional, I can't remember.